_docs-v5/view-api/view-render-hooks.md

```diff
@@ -18,3 +18,14 @@ When the above hooks are specified as a function in the form `function(arg)`, th
 
 - `view` - a [View Object](view-object)
 - `el` - the view's root HTML element. not available in `viewClassNames`
+
+
+## View Changes
+
+There are 2 general rules that drives the call fo the above mentioned hooks:
```

Comment:
`drives the call` is better said as `initiate`

also "the call **fo** the above" is wrong